config/config.yaml

```diff
@@ -64,3 +64,8 @@ tunings:
     wait_duration: 60                                      # Duration to wait between each chaos scenario
     iterations: 1                                          # Number of times to execute the scenarios
     daemon_mode: False                                     # Iterations are set to infinity which means that the kraken will cause chaos forever
+telemetry:
+    enabled: True
```

Comment:
We might want to disable it by default given that it's authenticated and users might not have the credentials.